server/s3_post_complete_action.cc

```diff
@@ -155,6 +149,29 @@ void S3PostCompleteAction::fetch_object_info_failed() {
   s3_log(S3_LOG_DEBUG, stripped_request_id, "Exiting\n");
 }
 
+void S3PostCompleteAction::fetch_ext_object_info_failed() {
+  s3_log(S3_LOG_DEBUG, request_id, "%s Entry\n", __func__);
+  if (object_metadata && object_metadata->is_object_extended()) {
```

Comment:
It seams that send_response_to_s3_client() must be moved outside the condition

Reply:
ok. This condition is true for such failed callback, but yes will remove if condition to avoid "no response to client" situation